src/XRP/issued-currency-client.ts

```diff
@@ -192,6 +202,33 @@ export default class IssuedCurrencyClient {
     }
   }
 
+  /**
+   * Subscribes to an account's incoming transactions, and triggers a callback upon recieving each transaction.
+   * @see https://xrpl.org/monitor-incoming-payments-with-websocket.html
+   * @see https://xrpl.org/subscribe.html
+   *
+   * @param account The account from which to subscribe to incoming transactions, encoded as an X-Address.
+   * @param callback The function to trigger upon recieving each transaction from the ledger.
+   * @returns The response from the websocket confirming the subscription.
```

Comment:
```suggestion
   * @returns The response from the websocket indicating the result of the subscription request.
```

Reply:
I'm wondering if it might not make sense to process the response a little bit in this method instead of exposing the raw `WebSocketStatusResponse`?  for example, returning just an indicator of success or failure... I don't know, that might cause some information loss such that an alternative solution wouldn't really be any cleaner than what we already have...

Reply:
I wasn't really sure what to return tbh - I could just return the status string. There isn't much else returned in the status object, so it's fine. 
```interface WebSocketStatusResponse {
  id: string
  result: WebSocketTransactionResponse | EmptyObject
  status: string
  type: string
}

Reply:
i do think it's a little weird that we're exposing some details about the implementation here. specifically, we're returning a `WebSocketTransactionResponse`, but i would expect `IssuedCurrencyClient` to abstract away the concept of `WebSocket`, if that make sense

Reply:
Unless I just change the name of `WebSocketTransactionResponse`, I'm not really sure how to fix that, because you _are_ expecting the whole transaction response object.

Reply:
Wait I don't know if we're talking about the same thing here:  we're looking at `monitorAccountTransactions` right? this method just needs to return an indication of success around establishing the subscription... the `WebSocketTransactionResponse` will be delivered to the registered callback upon transaction events... so that's a different thing entirely.  I think this method, `monitorAccountTransactions` could probably just return the status string `success`?  Or an error?  (Returning a string still seems a little weird?  is there a better or more conventional way to indicate success?)

Reply:
I was thinking I could just return `true` maybe? (or whether the status is `'success'`)